src/Symfony/Component/Config/Tests/Util/XmlUtilsTest.php

```diff
@@ -63,11 +63,14 @@ public function testLoadFile()
 
     public function testLoadFileWithInternalErrorsEnabled()
     {
-        libxml_use_internal_errors(true);
+        $internalErrors = libxml_use_internal_errors(true);
 
         $this->assertSame(array(), libxml_get_errors());
         $this->assertInstanceOf('DOMDocument', XmlUtils::loadFile(__DIR__.'/../Fixtures/Util/invalid_schema.xml'));
         $this->assertSame(array(), libxml_get_errors());
+
+        libxml_clear_errors();
```

Comment:
If the assertion or something else fails this part is never reached.
You eg. need to wrap this is in a catch or finally (Symfony 3.0) block to ensure it's always called.